src/python/grpcio/grpc/_cython/_cygrpc/credentials.pyx.pxi

```diff
@@ -338,8 +338,9 @@ def server_certificate_config_ssl(pem_root_certs, pem_key_cert_pairs):
 def server_credentials_ssl_dynamic_cert_config(initial_cert_config,
                                                cert_config_fetcher,
                                                bint force_client_auth):
-  if not isinstance(initial_cert_config, grpc.ServerCertificateConfig):
-    raise TypeError('initial_cert_config must be a grpc.ServerCertificateConfig')
+  if not isinstance(initial_cert_config, grpc.ServerCertificateConfiguration):
+    raise TypeError(
+        'initial_cert_config must be a grpc.ServerCertificateConfiguration')
```

Comment:
nit: we say `initial_cert_config` here where as the user is passing `initial_certificate_configuration` to while calling `dynamic_ssl_server_credentials()`. i guess it's close enough that they can figure it out? or, should we move this type check one layer up, into `__init__.py`?

Reply:
Thanks for noticing this.

In a separate change we should move this type check one layer up.